python/ql/src/Security/CWE-327/InsecureDefaultProtocol.qhelp

```diff
@@ -34,20 +35,42 @@
           <p>
             Both of the cases above should be updated to use a secure protocol
             instead, for instance by specifying
-            <code>ssl_version=PROTOCOL_TLSv1_1</code> as a keyword argument.
+            <code>ssl_version=PROTOCOL_TLSv1_2</code> as a keyword argument.
+          </p>
+          <p>
+            The latter example can also be made secure by modifying the created
+            context before it is used to create a connection and so will not be
+            flagged by this query. However, if a connection is created before
```

Comment:
It's not clear to me from this text whether we mean the second example is never flagged by the query because it might be used in a secure way, or the second example is not flagged by the query if it is used in a secure way.

Reply:
It is the first option. This query does not use data flow, so it now only flags cases that are immediately insecure. The other query referred to in the next sentence, `py/insecure-protocol`, uses data flow to detect if a context is made secure before it is used.

Reply:
I have attempted a clarification now.